python/common_validations.py

```diff
@@ -0,0 +1,28 @@
+import string_utils
+from parameters_validation import parameter_validation
+import re
+
+
+@parameter_validation
+def is_version(version: str):
+    if version is None or not version:
```

Comment:
can't we simply check `if not version:` ?

Reply:
Fixed by https://github.com/citusdata/tools/pull/91/commits/a2e8aef98e41c785588ddb62c2ed0f902bee180d